src/main/resources/logback.xml

```diff
@@ -82,16 +82,18 @@
   <!--
     | Setup default log level to INFO
   +-->
-  <root level="INFO">
+  <root level="WARN">
     <appender-ref ref="LOG" />
   </root>
 
   <!--
     | Turn up logging for portlet specific package
    +-->
-  <!--
   <logger name="edu.wisc.my.portlets.bookmarks" additivity="false" level="DEBUG">
```

Comment:
Should this be commented out since its level is `DEBUG`?

Reply:
We haven't really decided what default log level we want as a default; however, WARN sounds good with DEBUG/INFO commented out is a great suggestion.